SECURITY.md

```diff
@@ -0,0 +1,45 @@
+# Security
+
+This document specifies the security process for the SwiftNIO project.
+
+## Disclosures
+
+### Private Disclosure Process
+
+The SwiftNIO core team asks that known and suspected vulnerabilities be
+privately and responsibly disclosed by emailing
+[cve@forums.swift.org](mailto:cve@forums.swift.org) with the [details usually
```

Comment:
I assume you've checked this address is ok for us to use.

Reply:
Who actually gets those mails?

Reply:
I assume @tomerd -- but not certain

Reply:
the swift core team has access to this. we are considering creating such email alias for sswg projects which may be applicable here. cc @weissi

Reply:
Yeah, a NIO specific alias that posts to a private forums area would be great actually I think. WDYT other NIO devs?

Reply:
I think that's a good idea.

Reply:
@tomerd is it possible to get a NIO-specific one? @Lukasa / @glbrntt are we waiting for this or can we just merge this PR (I'm in favour of merging now).

Reply:
I reckon we use the sswg-security-reports one and merge. wdyt?

Reply:
Fine to merge now, we can always update later if/when we get a NIO specific one.

Reply:
I am somewhat uncomfortable with keeping these two addresses aliased together but that doesn't need to block the merge of this patch.

Reply:
this should be changes to `sswg-security-reports` (SSWG) which has more relevant people on it than `cve@forums.swift.org` (swift core team)